lib/practice.md

```diff
@@ -3,58 +3,61 @@
 - What package did you install for enumeration?
 
 ```md
-<!-- your answer here -->
+lodash
 ```
 
 ## Research: Popular NPM Packages
 
 Package 1:
 ```md
-<!-- your answer here -->
+express
 ```
 1.  What problem does this package solve?
 ```md
-<!-- your answer here -->
+routing
```

Comment:
This is an entire web application framework, but I wouldn't say that it solves a problem with routing.